docs/proposals/docker-event-collection.md

```diff
@@ -0,0 +1,86 @@
+# Docker Event Collection
+## Overview
+Docker events are not collected or stored by the Openshift cluster.  Certain customers are [required](https://trello.com/c/qxoFsKz9/) to audit these events (e.g. security).  This document is the architecture and design proposal for gathering Docker events for persistence.
```

Comment:
Consider changing last few words to, "for gathering Docker events for persistence as log entries interleaved with other operational logs."